client-service/rrv6/src/main/java/com/nortal/jroad/client/rrv6/RrV6XTeeServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+package com.nortal.jroad.client.rrv6;
+
+
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR441Document;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR441RequestType;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR442Document;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR442RequestType;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR442ResponseDocument;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR50SurnudIsikuteLeidmineDocument;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR50SurnudIsikuteLeidmineResponseDocument;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RRARKJUHILUBADocument;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RRARKJUHILUBAResponseDocument;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RRSIDEAADRESSSideDataDocument;
+import com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RRSIDEAADRESSSideDataResponseDocument;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.nortal.jroad.client.exception.XRoadServiceConsumptionException;
+
+
+@Service("Rrv6XTeeServiceImpl")
+public class RrV6XTeeServiceImpl implements RrV6XTeeService {
+
+    @Autowired
+    private com.nortal.jroad.client.rrv6.database.RrXRoadDatabase rrXRoadDatabase;
+    
+    @Override
+    public com.nortal.jroad.client.rrv6.types.eu.x_road.rr.producer.RR441ResponseDocument.RR441Response findRr441(String isikukood, String userId) throws XRoadServiceConsumptionException {
+
+        if (isikukood == null) {
+            throw new RuntimeException("Isikukood peab olema määratud!");
```

Comment:
The code is expected to be in English as much as possible. Please ensure all messages are in English. 
Unfortunately no coding rules have been defined yet.

It would be better to use more specific exception, such as IllegalArgumentException in case the argument value null is provided